src/gh/diffCheck/diffCheck/df_error_estimation.py

```diff
@@ -0,0 +1,156 @@
+#! python3
+"""
+    This module contains the utility functions to compute the difference between source and target
+"""
+
+import numpy as np
+from diffCheck import diffcheck_bindings
+import Rhino.Geometry as rg
+
+
+def cloud_2_cloud_comparison(source_list, target_list):
+    """
+        Compute the Euclidean distance for every point of a source pcd to its
+        closest point on a target pointcloud
+    """
+    results = DFVizResults()
+    for source, target in zip(source_list, target_list):
+        distances = np.asarray(source.compute_distance(target))
+        results.add(source, target, distances)
+
+    return results
+
+
+def cloud_2_rhino_mesh_comparison(cloud_source_list, rhino_mesh_target_list, signed_flag, swap):
+    """
+        Computes distances between a pcd and a mesh
+    """
+    results = DFVizResults()
+
+    for source, target in zip(cloud_source_list, rhino_mesh_target_list):
+        if swap:
+            # this mean we want to vizualize the result on the target mesh
+            distances = rhino_mesh_2_cloud_distance(target, source, signed_flag)
+        else:
+            # this means we want to vizualize the result on the source pcd
+            distances = cloud_2_rhino_mesh_distance(source, target, signed_flag)
+
+        if swap:
+            results.add(target, source, distances)
+        else:
+            results.add(source, target, distances)
+
+    return results
+
+
+# def cloud_2_mesh_distance(source, target, signed=False):
+#     """
+#         Calculate the distance between every point of a source pcd to its closest point on a target DFMesh
+#     """
+
+#     # for every point on the PCD compute the point_2_mesh_distance
+#     if signed:
+#         distances = np.asarray(target.compute_distance(source, is_abs=False))
+#     else:
+#         distances = np.asarray(target.compute_distance(source, is_abs=True))
+
+#     return distances
```

Comment:
Do you think this can be deleted, or could it come handy later ?

Reply:
same question, I think it is linked to the eventual cleaning of the backend+wrap

Reply:
deleted in the spirit of cleaning things up